storybookjs/storybook
# UI: Add version release notes

# This depends on https://github.com/storybookjs/frontpage/pull/107

Issue: #11232 

## What I did

This PR sets up the release notes logic in the build process & also in the ui.

## How to test

- **Is this testable with Jest or Chromatic screenshots?**

Yeah. I added some stories for the release notes page states, though I left the iframe out as that will likely change quite often.

- **Does this need a new example in the kitchen sink apps?**

Not sure. I added a new CLI flag for disabling the auto-messaging around release notes. Does that warrant a new example?

- **Does this need an update to the documentation?**

There is a new CLI flag, so that should be documented. Does that happen automatically though? I couldn't quite tell. I added it to `lib/core/src/server/cli/dev.js` with a note, which seemed to be documentation, but not sure where that shows up.
